charts/dnsmadeeasy-webhook/OWNERS

```diff
@@ -0,0 +1,4 @@
+approvers:
```

Comment:
We don't use the OWNERS file anymore in our repo. We have migrated to a repo-wide `CODEOWNERS` file.

Reply:
removed